mxcube3/routes/Utils.py

```diff
@@ -95,12 +97,142 @@ def my_execute_entry(self, entry):
         foo = ['collectOscillationFinished', 'collectOscillationFailed', 'warning']
         mxcube.collect.emit(random.choice(foo))
     elif isinstance(entry, qe.CharacterisationGroupQueueEntry):
+        logging.getLogger('HWR').info('[QUEUE] Executing CharacterisationGroupQueueEntry entry: %s' %entry)
```

Comment:
Sorry for not being clear... No, it is not used, it was used only in the beginning. You can ignore `my_execute_entry` completely.

I used this in the beginning when we did not have all the hwobj in place, so at least we could run and test client-server interaction without worrying about any hwobj. The sleep calls were for simulating an execution that takes time, otherwise we could not see the client switching task's colors.